packages/lib-classifier/src/helpers/index.js

```diff
@@ -1,8 +1,7 @@
 export { default as createLocationCounts } from './createLocationCounts'
-export { default as featureDetection } from './featureDetection'
```

Comment:
It turns out `featureDetection` doesn't have a default export, so I deleted it.

Reply:
Could you not? It's part of the service worker code I wrote, and although the service worker is not in use right now, if we're going to remove it, it should be removed altogether with that code where it's used.

Reply:
The `featureDetection` exports are still there. It’s just that `featureDetection` has no default export so this line is erroring in the console during module resolution.

Reply:
I'll change this to export all the named feature detection functions. That should fix the following warning on master.
```
WARNING in ./src/helpers/index.js 2:0-65
"export 'default' (reexported as 'featureDetection') was not found in './featureDetection'

```

Reply:
Ok, I misunderstood your first note as meaning you removed the helper function entirely. Thanks!